app/org/sagebionetworks/bridge/BridgeConstants.java

```diff
@@ -20,7 +20,14 @@
     
     public static final String BRIDGE_API_STATUS_HEADER = "Bridge-Api-Status";
 
-    public static final String BRIDGE_DEPRECATED_STATUS = "deprecated";
+    public static final String BRIDGE_DEPRECATED_STATUS = "you're calling a deprecated endpoint";
+
+    public static final String BRIDGE_WARNING_STATUS = "Bridge-Warning: you're calling a deprecated endpoint; you haven't included an Accept-Language header; we can't parse your User-Agent header";
+
+    public static final String WARN_NO_USER_AGENT =  "we can't parse your User-Agent header";
+
+    public static final String WARN_NO_ACCEPT_LANGUAGE = "you haven't included an Accept-Language header";
```

Comment:
messages used to include why we were warning people, we should add back (e.g. here we won't filter based on the caller's language)